lib/ruby_lsp/setup_bundler.rb

```diff
@@ -289,6 +289,7 @@ def rails_app?
       application_contents = config.read if config.exist?
       return false unless application_contents
 
+      application_contents = application_contents.encode(Encoding::UTF_8, Encoding::UTF_8, invalid: :replace)
```

Comment:
`force_encoding` would do the job as well but this is more robust against whatever else may be present in that file